orange_web/settings.py

```diff
@@ -119,4 +119,5 @@
 
 # Error report settings
 ERROR_REPORT_DIR = os.path.abspath("./error_report/")
-ERROR_REPORT_SENTRY_DSN = ''
+# Biolab's testing Sentry project
+ERROR_REPORT_SENTRY_DSN = 'https://c552df1b36cb4334a98ee4c37140af34:d5c900a3f3684d45b0ecc4aaeba540f1@sentry.io/127130'
```

Comment:
Nah, this is for a testing project, and can be easily revoked. DSNs for production projects are in a separate (private) file (see settings_production.py)